### DISCUSSION TITLE
Is Growth Management Area (GMA) a Universal Term

### DISCUSSION THREAD

**Discussion by penoptech** [Get-Your/Get-Your#163] [Q&A] [answered]
We use GMA throughout the `Get-Your` code base. I was wondering if that term is a universally used with all municipalities, or if that is something more unique to the City of Fort Collins. If it is more of a unique term for the City, is there more of a general term that means relatively the same thing for our use case?

**Comment by TiCampbell** [comment, marked as answer]
My understanding is that Growth Management Area (or at least Growth Management) is a pretty universal term, at least in the US. I think this is the best term to use for this.
